### PR TITLE
Add apply-width-height param to mixin

### LINKED ISSUE #10: `oIconsGetIcon` mixin with `$apply-base-styles: true` outputs width/height
In the `o-ft-icons`, when using the `oFtIconsGetSvg`, if the argument `$apply-base-styles` was set to `false`, _only_ the background image was output, i.e. not the `width` or `height`

https://github.com/Financial-Times/o-ft-icons/blob/master/scss/_mixins.scss#L47-L62

However, now using the `oIconsGetIcon` with `$apply-base-styles: true`, the `width` and `height` is output

https://github.com/Financial-Times/o-icons/blob/master/scss/_mixins.scss#L34-L42

(they're outside the conditional)

Was there a reason for this change? If not, can we go back to the `oFtIconsGetSvg` way?

### PR DESCRIPTION
**Pull request by onishiweb**
Fixes #10

### CODE REVIEW

**Comment by AlbertoElias** [comment]
Uhm, if you set the width variable to null, wouldn't it achieve the same effect? I'm still not saying we shouldn't merge this, as I'm not sure the other option is very user friendly

**Comment by onishiweb** [comment]
Just tried the calculation on that in Sassmeister and get the error: `Invalid null operation: "null plus 0px".` The current code only checks for `null` on the `height` not the width, so we'd have to do that to get the same result, I think this is possibly the best solution we'll get at the moment

File: scss/_mixins.scss [hunk 1]
```diff
@@ -31,11 +31,16 @@
 	// It still needs to use the build service  <-- what does this comment mean?
 	background-image: url($service-url + $query + "&format=png&width=#{$container-width}")\9;
 
-	width: $container-width + 0px;
-	@if ($container-height == null) {
-		$container-height: $container-width;
+	// Added so developers can prevent dimension styles from being output.
```
Comment (AlbertoElias) [inline scss/_mixins.scss]: I would remove the word developers from here

File: scss/_mixins.scss [hunk 2]
```diff
@@ -31,11 +31,16 @@
...
+	// This is because in the previous `oFtIconsGetSVG` mixin, the dimension
```
Comment (AlbertoElias) [inline scss/_mixins.scss]: Might be a hard to follow comment as you need to understand the history of the module and probably not necessary.

Comment (onishiweb) [inline scss/_mixins.scss]: Alice asked me to add the comment as otherwise I think the code looks a bit out of place. I'll reword it though.

**Comment by AlbertoElias** [comment]
👍

**Comment by onishiweb** [comment]
Just rebased the commits.
